content/docs/user-guide/scripting/_index.md

```diff
@@ -7,8 +7,8 @@ weight: 900
 
 **Open 3D Engine (O3DE)** includes two scripting languages for creating logic and behaviors: Script Canvas and Lua.
 
-**Script Canvas** is a visual scripting environment. In the Script Canvas editor, you create, connect, and rearrange graphical nodes that provide a visual representation of the logic flow. Script Canvas offers an approachable environment to author behaviors using the same framework as Lua and C++. You can use Script Canvas to create scripts without coding experience.
+**Script Canvas** is a general purpose, visual scripting environment. In the Script Canvas editor, you create, connect, and rearrange graphical nodes that provide a visual representation of the logic flow. Script Canvas offers an approachable environment to author behaviors using the same framework as Lua and C++. You can use Script Canvas to create scripts without coding experience.
```

Comment:
I like "layout" but I think "lay out" is the right spelling to use since it's a verb phrase. Also, practically-speaking, I think "create" and "lay out" are redundant when it comes to the Script Canvas editor.